command/acl/authmethod/formatter.go

```diff
@@ -58,6 +58,16 @@ func (f *prettyFormatter) FormatAuthMethod(method *api.ACLAuthMethod) (string, e
 		buffer.WriteString(fmt.Sprintf("DisplayName:  %s\n", method.DisplayName))
 	}
 	buffer.WriteString(fmt.Sprintf("Description:  %s\n", method.Description))
+	if method.MaxTokenTTL > 0 {
+		buffer.WriteString(fmt.Sprintf("MaxTokenTTL:  %s\n", method.MaxTokenTTL))
+	}
+	if len(method.NamespaceRules) > 0 {
+		buffer.WriteString(fmt.Sprintln("NamespaceRules:"))
```

Comment:
I feel like this should just error... if it's set? 
🤔

Reply:
I was keeping with how the namespace field is handled.